terraform/values.yaml

```diff
@@ -0,0 +1,521 @@
+command:
+  - "/opt/keycloak/bin/kc.sh"
+  - "start"
+  - "--auto-build"
+  - "--http-enabled=true"
+  - "--http-port=8080"
+  - "--hostname-strict=false"
+  - "--hostname-strict-https=false"
+extraEnv: |
+  - name: KEYCLOAK_ADMIN
+    value: ${KEYCLOAK_ADMIN_USERNAME}
+  - name: KEYCLOAK_ADMIN_PASSWORD
+    value: ${KEYCLOAK_ADMIN_PASSWORD}
+  - name: JAVA_OPTS_APPEND
+    value: >-
+      -Djgroups.dns.query={{ include "keycloak.fullname" . }}-headless
+  - name: DB_PORT
+    value: "${DB_PORT}"
+
+# Optionally override the fully qualified name
+fullnameOverride: ""
+
+# Optionally override the name
+nameOverride: ""
+
+# The number of replicas to create (has no effect if autoscaling enabled)
+replicas: 1
+
+image:
+  # The Keycloak image repository
+  repository: quay.io/keycloak/keycloak
+  # Overrides the Keycloak image tag whose default is the chart appVersion
+  tag: "latest"
```

Comment:
Can we change this to `18.0.0`? Just want to make sure it doesn't auto-upgrade seeing as KeyCloak upgrades usually require some downtime